entrypoint.sh

```diff
@@ -100,6 +100,10 @@ case "$1" in
     python "${@:2}"
   ;;
   * )
-    show_help
+    if [[ $2 == /opt/.pycharm_helpers/* ]]; then
+      ${@}
+    else
+      show_help
+    fi
```

Comment:
This allows PyCharm to use the Python interpreter from Iaso's Docker Compose. It's not related to the PR but is very useful to me.